docs/user-guide/persistent-volumes/index.md

```diff
@@ -222,6 +222,51 @@ A volume will be in one of the following phases:
 
 The CLI will show the name of the PVC bound to the PV.
 
+### Mount Options
+
+Kubernetes administrator can specify additional mount option to be used when a Persistent Volume is
```

Comment:
Okay. In any case it still has grammar issues. Suggested edit:

"A Kubernetes administrator can specify additional mount options for when a Persistent Volume is being mounted on a node."